shared-helpers/src/locales/general.json

```diff
@@ -234,7 +234,7 @@
   "application.household.membersInfo.title": "Before adding other people, make sure that they aren't named on any other application for this listing.",
   "application.household.preferredUnit.legend": "Preferred unit type",
   "application.household.preferredUnit.options.fiveBdrm": "5 Bedroom",
-  "application.household.preferredUnit.options.fourBdrm": "4 Bedroom",
+  "application.household.preferredUnit.options.fourBdrm": "4+ Bedroom",
```

Comment:
Question: If this change only is going to be in Detroit can this be added to 
`sites/public/page_content/locale_overrides/general.json` instead of this file?